src/OpenTelemetry.ResourceDetectors.Host/README.md

```diff
@@ -0,0 +1,39 @@
+# Host Resource Detectors
+
+[![NuGet version badge](https://img.shields.io/nuget/v/OpenTelemetry.ResourceDetectors.Host)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.Host)
+[![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.ResourceDetectors.Host)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.Host)
+
+## Getting Started
+
+You need to install the
+`OpenTelemetry.ResourceDetectors.Host` package to be able to use the
+Host Resource Detectors.
+
+```shell
+dotnet add package OpenTelemetry.ResourceDetectors.Host --prerelease
+```
+
+## Usage
+
+You can configure Host resource detector to
+the `TracerProvider` with the following example below.
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.ResourceDetectors.Host;
+
+var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                        // other configurations
+                        .ConfigureResource(resource => resource
+                            .AddDetector(new HostDetector()))
+                        .Build();
+```
+
+The resource detectors will record the following metadata based on where
+your application is running:
+
+- **HostDetector**: `host.name`.
```

Comment:
please link to https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/host.md, and warn about the non-stable nature of this convention.

Reply:
Please check the wording in: caa4f0477d71b8a4783ee80503ff870879dcbd2a

It will be rendered as:
<img width="755" alt="image" src="https://github.com/open-telemetry/opentelemetry-dotnet-contrib/assets/5972917/71c19c44-fb89-4e76-815f-3380d82e15aa">